src/main/java/edu/stanford/protege/webprotege/postcoordinationservice/repositories/PostCoordinationRepository.java

```diff
@@ -132,4 +134,74 @@ public Optional<EntityCustomScalesValuesHistory> getExistingCustomScaleHistoryOr
         );
 
     }
+
+    public void deletePostCoordinationCustomScalesRevision(ChangeRequestId changeRequestId, ProjectId projectId, String entityIri) {
+
+        Query query = new Query();
+        query.addCriteria(
+                Criteria.where(WHOFIC_ENTITY_IRI).is(entityIri)
+                        .and(PROJECT_ID).is(projectId.value())
+        );
+        Update update = new Update().pull("postCoordinationCustomScalesRevisions",
+                new Document("changeRequestId._id", changeRequestId.id()));
```

Comment:
[suggestion]: in database i think we should only store the actual changeRequestId and not the ChangeRequestId object.

Reply:
changed to string only.